### DISCUSSION TITLE
Sort Options for Collections

### DISCUSSION THREAD

**Discussion by kanjieater** [gotson/komga#1538] [Ideas]
### Describe your suggested feature

Currently in collections there are no sort options. Ideally collections would have the same sort options as series.
![image](https://github.com/gotson/komga/assets/32607317/1ebceee4-6a0b-4c86-aa2d-cc6050141eb0)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Collections are nothing like series, what kind of sort fields would you want, and why ?

**Comment by kanjieater** [comment]
Sometimes Collections can be large and sorting them helps to organize, find, or prioritize them. For instance I wanted to sort this list by Last Updated, but couldn't 
![image](https://github.com/gotson/komga/assets/32607317/69c48c89-01ff-47ab-b6da-fa4f26185e7a)


I would want all of the same sort options as series (including any future sort options like random & last read)

![image](https://github.com/gotson/komga/assets/32607317/56495069-03ac-4f3f-b3db-5ef1b5ba7677)

**Comment by KenyaDigIt2** [comment]
> Collections are nothing like series, what kind of sort fields would you want, and why ?

Can you explain how collections are supposed to work? I think that is where some confusion is coming from. I believed that collections where a way to group multiple series together. In that case sorting is a feature that would improve the program as a whole.

For example say I have a collection for Hellboy. It contains all the limited series, one shots, and other related books. The ideal way to read all of the books would be to sort the collection by released date and see each individual book in that order. As of now the only way to accomplish this is to manually order a reading list, which take a long time for those that have a lot of books.

**Comment by gotson** [comment]
> > Collections are nothing like series, what kind of sort fields would you want, and why ?
> 
> Can you explain how collections are supposed to work? I think that is where some confusion is coming from. I believed that collections where a way to group multiple series together. In that case sorting is a feature that would improve the program as a whole.
> 
> For example say I have a collection for Hellboy. It contains all the limited series, one shots, and other related books. The ideal way to read all of the books would be to sort the collection by released date and see each individual book in that order. As of now the only way to accomplish this is to manually order a reading list, which take a long time for those that have a lot of books.

a collection shows series. If you want to see the individual books from multiple series, you need to use a read list.

**Comment by KenyaDigIt2** [comment]
Is there a way to turn a collection into a reading list automatically?  That would solve the issue of not being able to sort collections since reading lists have a sort option.

**Comment by gotson** [comment]
> Is there a way to turn a collection into a reading list automatically?

no

**Comment by gotson** [comment]
Collections are either sorted by release date or manually, sort options within a collection for display don't make sense.

**Reply by kanjieater** [reply]
So if I made a collection of "Favorite Book Club Series 2024" it wouldn't make sense to sort them by the date they were added?

**Reply by gotson** [reply]
if you set the collection to manual ordering, that is what is happening, if you haven't changed the order manually.
